src/shared_modules/keystore/src/keyStore.cpp

```diff
@@ -10,10 +10,47 @@
  */
 
 #include "keyStore.hpp"
+#include "rsaHelper.hpp"
+#include "loggerHelper.h"
 
+constexpr auto KS_NAME {"keystore"};
 
-void Keystore::put(const std::string& columnFamily, const std::string& key, const rocksdb::Slice& value)
+namespace Log
 {
+    std::function<void(
+        const int, const std::string&, const std::string&, const int, const std::string&, const std::string&, va_list)>
+    GLOBAL_LOG_FUNCTION;
+};
+
+void Keystore::put(const std::string& columnFamily, const std::string& key, const std::string& value)
+{
+    std::string encryptedValue;
+
+    // Encrypt value
+    try
+    {
+        int encrypted_len = Utils::rsaEncrypt(CERTIFICATE_FILE, value, encryptedValue, true);
+    }
+    catch (std::exception& e)
+    {
+        logError(KS_NAME, "%s", e.what());
+    }
+
+    // Insert to DB
+    try
+    {
+        Utils::RocksDBWrapper keystoreDB = Utils::RocksDBWrapper(DATABASE_PATH, false);
+
+        if (!keystoreDB.columnExists(columnFamily)) {
+            keystoreDB.createColumn(columnFamily);
+        }
+
+        keystoreDB.put(key, rocksdb::Slice(encryptedValue), columnFamily);
+    }
+    catch (std::exception& e)
+    {
+        logError(KS_NAME, "%s", e.what());
+    }
```

Comment:
All statements should be included in a single `try-catch` block, to avoid continuing if the first block returns an error and to simplify the code.
The use of `auto` is also preferred.
```c++
    try
    {
        std::string encryptedValue;

        // Encrypt value
        auto encrypted_len = Utils::rsaEncrypt(CERTIFICATE_FILE, value, encryptedValue, true);

        // Insert to DB
        Utils::RocksDBWrapper keystoreDB = Utils::RocksDBWrapper(DATABASE_PATH, false);

        if (!keystoreDB.columnExists(columnFamily)) {
            keystoreDB.createColumn(columnFamily);
        }

        keystoreDB.put(key, rocksdb::Slice(encryptedValue), columnFamily);
    }
    catch (std::exception& e)
    {
        logError(KS_NAME, "%s", e.what());
    }
```